src/pyhf/contrib/viz/brazil.py

```diff
@@ -1,13 +1,255 @@
 """Brazil Band Plots."""
+from typing import List, NamedTuple
+
+import matplotlib
+import matplotlib.pyplot as plt
 import numpy as np
 
 
-def plot_results(ax, mutests, tests, test_size=0.05):
+class BrazilBandCollection(NamedTuple):
+    r"""
+    Tuple containing the :class:`matplotlib.artist.Artist` objects of the
```

Comment:
The docs are breaking when subclassing `typing.NamedTuple` and I'm not really sure why. @kratsg if you have time later to give thoughts here that could be useful.

Reply:
I gave up and used `collections.namedtuple`.